test/hpke_test.c

```diff
@@ -838,6 +838,16 @@ static char *bogus_suite_strs[] = {
     "bogus",
     /* in reverse order */
     "aes-256-gcm,hkdf-sha512,x25519",
+    /* delimiter diahorrea :-) */
```

Comment:
maybe remove this in case someone is offended..

Reply:
entirely willing to do that, but kinda linking the alliteration and aptness, I'll wait 'till tomorrow:-)